docs(framework): Update Node Authentication docs (#6009)
adap/flower: framework/docs/source/how-to-authenticate-supernodes.rst

        --ssl-ca-certfile certificates/ca.crt \
        --ssl-certfile certificates/server.pem \
        --ssl-keyfile certificates/server.key \
        --enable-supernode-auth

.. dropdown:: Understand the command

    * ``--enable-supernode-auth``: Enables SuperNode authentication therefore only Supernodes that are first register on the SuperLink will be able to establish a connection.

Register SuperNodes
-------------------

With a SuperLink already running, we can proceed to register the SuperNodes that will be
allowed to connect to it. This is done via the |flower_cli_supernode_link|_ using the
public keys generated earlier for each SuperNode that we want to eventually connect to

> the SuperLink. Let's see how to this looks in code:

Reviewer: ```suggestion
Once your SuperLink is running, the next step is to register the SuperNodes that will be
allowed to connect to it. This process is handled through the
|flower_cli_supernode_link|_ using the public keys previously generated for each
SuperNode you plan to connect to the SuperLink.

Here's how this looks in code:
```

.. code-block:: bash

    # flwr supernode register <supernode-pub-key> <app> <federation>
    $ flwr supernode register keys/client_credentials_1.pub . local-deployment

Let's proceed and also register the second SuperNode:

.. code-block:: bash

    $ flwr supernode register keys/client_credentials_2.pub . local-deployment

You can list the registered SuperNodes using the following command:

.. code-block:: bash

    # flwr supernode list <app> <federation>
    $ flwr supernode list . local-deployment

> which should display the IDs of the SuperNodes you just registered as well as their

Reviewer: ```suggestion
This will display the IDs of the SuperNodes you just registered as well as their
```

    ┏━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━┓
    ┃       Node ID        ┃   Owner    ┃ Status  ┃ Elapsed  ┃   Status Changed @   ┃
    ┡━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━┩
    │ 16019329408659850374 │ sys_noauth │ online  │ 00:00:30 │ 2025-10-13 13:40:47Z │
    ├──────────────────────┼────────────┼─────────┼──────────┼──────────────────────┤
    │ 8392976743692794070  │ sys_noauth │ online  │ 00:00:22 │ 2025-10-13 13:52:21Z │
    └──────────────────────┴────────────┴─────────┴──────────┴──────────────────────┘

Unregister SuperNodes
---------------------

At anypoint you can unregister a SuperNode from the SuperLink (even if it has never
connected). This will prevent the SuperNode from making future request to the SuperLink.
In other words, it will no longer be authorized to pull/send, or participate in ongoing
or future runs. Unregistering a SuperNode can be done via the
|flower_cli_supernode_link|_ as follows:

.. code-block:: bash

    # flwr supernode unregister <node-id> <app> <federation>
    $ flwr supernode unregister 16019329408659850374 . local-deployment

> The above command unregistered the first SuperNode. You can verify this by listing the

Reviewer: ```suggestion
The above command unregisters the first SuperNode. You can verify this by listing the
```

SuperNodes again:

.. code-block:: bash

    $ flwr supernode list . local-deployment

    ┏━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━┓
    ┃       Node ID        ┃   Owner    ┃ Status  ┃ Elapsed  ┃   Status Changed @   ┃
    ┡━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━┩
    │ 8392976743692794070  │ sys_noauth │ online  │ 00:00:22 │ 2025-10-13 13:52:21Z │
    └──────────────────────┴────────────┴─────────┴──────────┴──────────────────────┘

If you pass the ``--verbose`` flag to the previous command you'll see that the status of
the unregistered SuperNode has changed to ``unregistered``. By default unregistered

> SuperNodes aren't shown becuase they can no longer particpate or reconnect. That's

Reviewer: ```suggestion
SuperNodes are hidden because they can no longer reconnect to the SuperLink. That's
```
